routes/home.js

```diff
@@ -1,7 +1,21 @@
-const router = require('express').Router()
+const router = require("express").Router();
+const Product = require("../models/ProductModel");
 
-router.get('/', (req, res) => {
-    res.send("Home page")
-})
+router.get("/", (req, res) => {
+  res.send("Home page");
+});
 
-module.exports = router
\ No newline at end of file
+router.get("/products", async (req, res) => {
```

Comment:
1. Please move this to a separate file.
2. That file needs to be under a new folder called `examples`.
3. Name the new file as `paginationExample.js`. Also, please create POST route for creations of products. And if possible, all CRUD operations.
4. If you end up creating all operations, they need to be accessible only with auth token. To be more precise, Create, Update and Delete operations need to be accessed with auth-token header only. Please take a look at [privateRoutes file](https://github.com/udz-codes/express-jwt-boilerplate/blob/master/routes/privateRoutes.js#L4-L7) for that.
5. Please add them as a middleware in [app.js](https://github.com/udz-codes/express-jwt-boilerplate/blob/master/app.js#L21).

PS: Apologies for not including these requirements early on, these should have been mentioned clearly before. Will take care of that in future.

Reply:
No problem. Worked on the updates